Web/packages/content/src/hooks/useGestureRecognizer.ts

```diff
@@ -81,7 +81,8 @@ export function useGestureRecognizer(
     const g = getClosestGesture();
     onRelease(g);
     patternConstructor.current.clear();
-  }, [onRelease, getClosestGesture]);
+    document?.getSelection()?.removeAllRanges();
+  }, [patternConstructor.current]);
```

Comment:
It looks like you deleted `onRelease, getClosestGesture` and included `patternConstructor.current` to the dependencies array. Is this intentional?

There was a version mismatch, and eslint is not working correctly, but `eslint-plugin-react-hooks` shows the following warning:

<img width="634" alt="Screen Shot 2022-08-15 at 8 14 15" src="https://user-images.githubusercontent.com/10496155/184558518-c962fb6d-9dad-4eb3-8c11-e00ad546f095.png">

Reply:
oh, sorry, its mistake... (I don't have intention